File: Back/src/ProEventos.Domain/Models/Palestrante.cs
using System.ComponentModel.DataAnnotations.Schema;
using ProEventos.Domain.Dtos;
using ProEventos.Domain.Identity;

namespace ProEventos.Domain
{
    [Table("palestrantes")]
    public class Palestrante
    {
        public int Id { get; set; }
        public string MiniCurriculo { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public IEnumerable<RedeSocial> RedeSociais { get; set; }
        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}

File: Front/ProEventos.APP/src/app/models/Palestrante.ts
import { User } from 'src/app/models/Users/User';
import { Evento } from "./Evento";
import { RedeSocial } from "./RedeSocial"

export interface Palestrante {
  id: number;
  user: User;
  miniCurriculo: string;
  redeSociais: RedeSocial[];
  PalestrantesEventos: Evento[];
}
